subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidPluginsSmokeTest.groovy

```diff
@@ -135,7 +138,7 @@ class AndroidPluginsSmokeTest extends AbstractSmokeTest {
 
         where:
         [agpVersion, ide] << [
-            TestedVersions.androidGradle.toList(),
+            TestedVersions.androidGradle.toList().takeRight(2).take(1),
```

Comment:
❌ I don't get this, could you maybe extract an explaining method of the intention here?

Reply:
Good catch. It's just a remnant of local testing. Removed